client/index.html

```diff
@@ -35,6 +35,7 @@
 		</aside>
 		<footer id="footer">
 			<span class="tooltipped tooltipped-n" aria-label="Sign in"><button class="icon sign-in" data-target="#sign-in" aria-label="Sign in"></button></span>
+			<span class="tooltipped tooltipped-n" aria-label="Sign up"><button class="icon sign-up" data-target="#sign-up" aria-label="Sign up"></button></span>
```

Comment:
Since you only show the `sign up` button if enabled (on line 102), it should probably also be done here